lib/sparkql/parser.y

```diff
@@ -88,11 +90,18 @@ rule
 # Keyword for searching on, these fields should be discovered using the metadata 
 # rules. In general, Keywords that cannot be found will be dropped from the 
 # filter.
+
   field
-  	: STANDARD_FIELD
-  	| CUSTOM_FIELD
-  	| function
-  	;
+    : field ADD field { result = tokenize_arithmetic(val[0], val[1], val[2]) }
```

Comment:
This doesn't sit right with me. I'd extract an `arithmetic_expression` terminal out of this or incorporate this with "field" some other way. I guess I started this with `function` and I apologize.

In any case, a bunch of these things are no longer fields, and we should update / doc the terminal accordingly.

Reply:
👍 Yeah, that sounds good to me.